packages/authx/src/model/Authorization.ts

```diff
@@ -381,13 +391,6 @@ export class Authorization implements AuthorizationData {
       return [];
     }
 
-    if (options?.forUpdate) {
```

Comment:
Was this delete intentional? It seems like this will cause the 404 issue to happen if two clients try to disable the same authorization at the same time (granted, sort of unlikely).

Reply:
Oh, no - great catch! That must have been a mistake while rebasing; I will fix this.

Reply:
OK, just rebased that commit. Great catch :)